src/sentry/processing/realtime_metrics/base.py

```diff
@@ -97,6 +98,14 @@ def get_lpq_projects(self) -> Set[int]:
         """
         raise NotImplementedError
 
+    def is_lpq_project(self, project_id: int) -> bool:
```

Comment:
this probably needs to be added to the `TYPE_CHECKING` block in `realtime_metrics/__init__.py`